Dockerfile

```diff
@@ -329,36 +328,45 @@ COPY airflow/www/ ${AIRFLOW_SOURCES}/airflow/www/
 # Package NPM for production
 RUN npm run prod
 
-# Cache for this line will be automatically invalidated if any
-# of airflow sources change
-COPY . ${AIRFLOW_SOURCES}/
-
-WORKDIR ${AIRFLOW_SOURCES}
-
-# Finally install the requirements from the latest sources
-RUN pip install -e ".[${AIRFLOW_EXTRAS}]"
-
-# Additional python deps to install
-ARG ADDITIONAL_PYTHON_DEPS=""
-
-RUN if [[ -n "${ADDITIONAL_PYTHON_DEPS}" ]]; then \
-        pip install ${ADDITIONAL_PYTHON_DEPS}; \
-    fi
-
 COPY ./scripts/docker/entrypoint.sh /entrypoint.sh
 
 COPY .bash_completion run-tests-complete run-tests ${HOME}/
+
 COPY .bash_completion.d/run-tests-complete \
      ${HOME}/.bash_completion.d/run-tests-complete
 
-COPY license-templates ${AIRFLOW_SOURCES}/license-templates
-
 RUN echo ". ${HOME}/.bash_completion" >> "${HOME}/.bashrc"
 
 RUN chmod +x "${HOME}/run-tests-complete"
 
 RUN chmod +x "${HOME}/run-tests"
 
+# Copy selected subdirectories only
+COPY .github/ ${AIRFLOW_SOURCES}/.github/
+COPY dev/ ${AIRFLOW_SOURCES}/dev/
```

Comment:
Do we need this folder in the image?

Reply:
dev probably not. I can remove it and see if it still works